### ISSUE TITLE
Add Perses / OpenFE (and potentially alchemiscale) to the list of tools mentioned

### ISSUE THREAD

**Issue by apayne97** [alchemistry/alchemical-best-practices#128]
I know this was not put in the original manuscript because it wasn't ready at the time but I think pretty soon we should consider adding it!

**Comment by ppxasjsm** [comment]
yes totally agree. We can add stuff now. There should be an updated version next year or so.

**Comment by fjclark** [comment]
Closed via the Overleaf updates to the software section, as discussed in https://github.com/alchemistry/alchemical-best-practices/issues/153. Thanks!
